test/api/dockerflow.test.ts

```diff
@@ -22,7 +22,9 @@ describe('dockerflow endpoints', () => {
 
   function setupForHeartbeat({
     failBitly,
-  }: $Shape<{| +failBitly: boolean |}> = {}) {
+  }: Partial<{
+    failBitly: boolean;
+  }> = {}) {
     let nockScope = nock(BITLY_HOSTNAME, {
```

Comment:
I don't see types for `nock`, is that expected?

update: I see that nock comes with its own types.